### ISSUE TITLE
Suggestion: Cop for missing doc for a given param

### ISSUE THREAD

**Issue by renatolond** [ksss/rubocop-yard#12]
This is a very common issue in our team: we add a new param to a function and forget to add documentation for it.
If we just rename the param, most likely it is caught by `YARD/MismatchName`, but in this case it is not.

What I would like is that there was a cop that would alert about a param without corresponding documentation, like I suggested originally in https://github.com/rubocop/rubocop/issues/11933

So in this case:

```
# frozen_string_literal: true

# @param [String] strings
def foo(strings, opts = {})
...
```

We would get a warning about `opts` not having corresponding documentation

**Comment by ksss** [comment]
Finding parameters that were forgotten in the documentation seems like a very good suggestion. However, simply implementing this proposal would force YARD documentation for every method that has any arguments, which seems a bit too stringent.

A few ideas come to mind:

- Enable this suggestion only if either `@param` or `@option` exists.
- It's disabled by default, but add an EnforcedStyle for strong enforcement.

Would these ideas work for your team?

**Comment by renatolond** [comment]
I think the first idea, of enabling only if there's some documentation already, would work very well for us 😄 . We can use https://docs.rubocop.org/rubocop/cops_style.html#styledocumentationmethod to require documentation and this would complete it by alerting about missing arguments 👍🏼

**Comment by ksss** [comment]
I'm glad to hear that.
I'll think about the implementation.

**Comment by ksss** [comment]
@renatolond 
I've implemented this feature and released it as v0.7.0.
Please check it out!

**Comment by renatolond** [comment]
We've been testing and it seems to work perfectly, thanks a lot once again 🙏🏼
